Svc/Interfaces/FramerInterface.fppi

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------
+# Framing
+# ----------------------------------------------------------------------
+@ Port to receive data to frame, in a Fw::Buffer with optional context
+guarded input port dataIn: Svc.ComDataWithContext
```

Comment:
Why is this one port "guarded" and everything else isn't.  I'd argue that it should be `sync` and if a user needs other behavior, they can lock.

Typically a framer should not be dealing in stored state.  It is all context and input data.

Reply:
Fix in this PR.